assets/js/blocks/cart-checkout/checkout/attributes.ts

```diff
@@ -5,56 +5,10 @@ import { getSetting } from '@woocommerce/settings';
 
 export const blockName = 'woocommerce/checkout';
 export const blockAttributes = {
-	isPreview: {
-		type: 'boolean',
-		default: false,
-		save: false,
-	},
 	hasDarkControls: {
 		type: 'boolean',
 		default: getSetting( 'hasDarkEditorStyleSupport', false ),
 	},
-	showCompanyField: {
-		type: 'boolean',
-		default: false,
-	},
-	requireCompanyField: {
-		type: 'boolean',
-		default: false,
-	},
-	allowCreateAccount: {
-		type: 'boolean',
-		default: false,
-	},
-	showApartmentField: {
-		type: 'boolean',
-		default: true,
-	},
-	showPhoneField: {
-		type: 'boolean',
-		default: true,
-	},
-	requirePhoneField: {
-		type: 'boolean',
-		default: false,
-	},
-	// Deprecated - here for v1 migration support
```

Comment:
Should we leave these out of metadata and possible put them with the block registration function so it's easier to remove them in future?

Reply:
I'm not sure, would it really make it much easier? I see we lose the comment when moving it to metadata, but besides that what would be easier about leaving them in the block registration function?

Reply:
The comment as you say, with a `deprecated` docblock. Some are now split between this file and metadata, so it's going to be hard to remove them in the future unless it's documented somewhere. Not major.

Reply:
Right, I've moved it to `deprecatedAttributes` in `attributes.ts` this seems to make more sense and makes it more obvious they're deprecated.